qiskit/ignis/verification/accreditation/fitters.py

```diff
@@ -20,12 +20,15 @@
 Samuele Ferracin, Theodoros Kapourniotis and Animesh Datta
 New Journal of Physics, Volume 21, November 2019
 https://iopscience.iop.org/article/10.1088/1367-2630/ab4fd6
+as well as Ferracin, Merkel, McKay and Datta
```

Comment:
It's better to write specifically:
References:
[1] Samuele Ferracin,...
[2] Samuele Ferracin,...
and also add links to the papers.
see an example in the clifford class.